File: 프로그래머스/0/181901. 배열 만들기 1/배열 만들기 1.cs
using System;
using System.Collections.Generic;

public class Solution {
    public int[] solution(int n, int k) {
        List<int> numbers = new List<int>();
        int i = 1;
        while(k * i <= n) {
            numbers.Add(k * i);
            i++;
        }
        
        int[] answer = numbers.ToArray();
        return answer;
    }
}

File: 프로그래머스/0/181901. 배열 만들기 1/배열 만들기 1.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(int n, int k)
{
    vector<int> v;
    for(int i = 1; i<=n; i++)
    {
        if(i%k == 0)
        {
            v.push_back(i);
        }
    }
    return v;
}